examples/advection_1d/advection_1d.py

```diff
@@ -45,13 +47,16 @@ def setup(nx=100, kernel_language='Python', use_petsc=False, solver_type='classi
     else: raise Exception('Unrecognized value of solver_type.')
 
     solver.kernel_language = kernel_language
-
+    solver.order=1
```

Comment:
Why are you removing the 2nd order correction and limiters here?

Reply:
I think because it was complaining about how None is not a limiter and not running.